pkg/kubelet/dockershim/libdocker/fake_client.go

```diff
@@ -565,6 +578,18 @@ func (f *FakeDockerClient) StartContainer(id string) error {
 	}
 	f.appendContainerTrace("Started", id)
 	container, ok := f.ContainerMap[id]
+	if container.HostConfig.PidMode.IsContainer() {
```

Comment:
Why `PidMode`?

Reply:
That is basically just asking if the container is attaching to another container's network namespace.
https://github.com/moby/moby/blob/master/api/types/container/host_config.go#L110

Reply:
It should not be pidmode? right?
Should be network mode.

Reply:
that would make more sense

Reply:
doen

Reply:
done